plugins/rtmp-services/data/services.json

```diff
@@ -3425,6 +3425,9 @@
             "common": false,
             "more_info_link": "https://docs.dolby.io/streaming-apis/docs/using-obs",
             "stream_key_link": "https://streaming.dolby.io",
+            "multitrack_video_configuration_url": "https://api.millicast.com/api/multitrackvideo/configure",
+            "multitrack_video_name": "Enhanced Broadcasting",
+            "multitrack_video_learn_more_link": "https://docs.dolby.io/streaming-apis/docs/obs-enhanced-broadcasting-multitrack-video",
```

Comment:
this link gives me a 404 Not Found error

Reply:
link is live now